apps/dashboard/src/components/workflow-editor/steps/email/maily.tsx

```diff
@@ -123,6 +127,13 @@ export const Maily = ({ value, onChange, className, ...rest }: MailyProps) => {
         variables: calculateVariables,
         variableSuggestionsPopover: MailyVariablesList,
       }),
+      HTMLCodeBlockExtension.extend({
+        addNodeView() {
+          return ReactNodeViewRenderer(HTMLCodeBlockView, {
+            className: 'mly-relative',
```

Comment:
We should use our own tailwind classes, maily's are not dynamic, they're generated on build time.

Reply:
We can remove all the `mly-` classes in separate PR.